python/fledge/services/core/api/control_service/acl_management.py

```diff
@@ -147,19 +155,29 @@ async def add_acl(request: web.Request) -> web.Response:
 
 @has_permission("admin")
 async def update_acl(request: web.Request) -> web.Response:
-    """ Update an access control list. Only the set of service and URL's can be updated
```

Comment:
This is as per specs
```
Update an ACL
PUT /fledge/ACL/{name}
Update an access control list. Only the set of services and URL’s in the ACL can be updated, this call can not be used to change the name or users of an ACL.
```
Not sure from where the requirement comes from for changing the name of ACL

Also check schema for ACL table we have a primary constraint for name as well. https://github.com/fledge-iot/fledge/blob/develop/scripts/plugins/storage/sqlite/init.sql#L617

cc/ @MarkRiddoch

Reply:
Also, the same case for Control script?

I can understand, that we may need to propagate changes to services/scripts on ACL rename. Should we keep this static then in GUI? or later add a different endpoint with propagation to references? 

Flexibility to fix naming is very basic to have.